test/EVMHost.cpp

```diff
@@ -449,11 +447,11 @@ evmc::result EVMHost::precompileECRecover(evmc_message const& _message) noexcept
 			}
 		}
 	};
-	evmc::result result = precompileGeneric(_message, inputOutput);
+	evmc::Result result = precompileGeneric(_message, inputOutput);
 	// ECRecover will return success with empty response in case of failure
 	if (result.status_code != EVMC_SUCCESS && result.status_code != EVMC_OUT_OF_GAS)
-//		return resultWithGas(_message.gas, gas_cost, {});
 	{
+		// Could use resultWithGas(_message.gas, gas_cost, {}) here, but it expects a reference.
 		result.status_code = EVMC_SUCCESS;
 		result.gas_left = _message.gas - gas_cost;
 		result.output_data = {};
```

Comment:
I'd actually have made this
```suggestion
		result.output_data = nullptr;
```
to make it clearer what this is, independently of https://github.com/ethereum/solidity/pull/13532/files#r1017828619

Reply:
Fixed.